src/main/scala/nl.knaw.dans.easy.archivebag/EasyArchiveBag.scala

```diff
@@ -67,7 +68,7 @@ object EasyArchiveBag extends Bagit5FacadeComponent with DebugEnhancedLogging {
       case HttpStatus.SC_UNAUTHORIZED =>
         throw UnautherizedException(ps.bagId)
       case _ =>
-        throw new RuntimeException(s"Bag archiving failed: ${ response.getStatusLine }")
+        throw new RuntimeException(s"Bag archiving failed: ${ps.storageDepositService.toString} returned:[ ${ response.getStatusLine } ]. ZippedBag=$zippedBag")
```

Comment:
@lindareijnhoudt Are the tmp files cleaned up after a while?

Reply:
yes